cmd/disk-cache.go

```diff
@@ -44,7 +44,6 @@ const (
 	// disk cache needs to have cacheSizeMultiplier * object size space free for a cache entry to be created.
 	cacheSizeMultiplier  = 100
 	cacheTrashDir        = "trash"
-	cacheMaxDiskUsagePct = 80 // in %
```

Comment:
`cacheMaxDiskUsagePct` default value is not saved anywhere, what is the default value to begin with?

Reply:
Then the default value should be  `globalCacheMaxUse = 80`,  changing it to 100 is not correct. We should also perhaps throw a warning but will get some confirmation on that.

Reply:
Addressed.